controllers/template_controller.go

```diff
@@ -99,7 +100,7 @@ func (r *TemplateReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		return reconcile.Result{}, err
 	}
 	incSuccess(name)
-	return ctrl.Result{}, nil
+	return ctrl.Result{Requeue: true, RequeueAfter: 2 * time.Minute}, nil
```

Comment:
Why the re-queue ? This would put an extreme load on both the API server and the operator?

Reply:
Since the test passes with expected logs so removing the requeue... I might be missing on what triggers the reconcile loop again for any given object